pkg/util/expectations/scale_expectations.go

```diff
@@ -35,7 +37,8 @@ func NewScaleExpectations() ScaleExpectations {
 type realScaleExpectations struct {
 	sync.Mutex
 	// key: parent key, workload namespace/name
-	controllerCache map[string]*realControllerScaleExpectations
+	controllerCache          map[string]*realControllerScaleExpectations
+	lastUnsatisfiedTimestamp time.Time
```

Comment:
Should not define `lastUnsatisfiedTimestamp ` here, because `realScaleExpectations` is a global cache.